### PR TITLE
completed p0-Wishlist page

### PR DESCRIPTION
**Pull request by vidhi-thakur**
created a wishlist page with navbar, heading and cards with icons

image

check the preview [here](https://deploy-preview-3--savvy-store.netlify.app/pages/wishlist.html )

### CODE REVIEW

File: page-css/wishlist.css [hunk 1]
```diff
@@ -0,0 +1,12 @@
+.wishlist-heading {
+    margin: 1.5rem auto 0;
+    width: fit-content;
+}
+
+.wishlisted-items {
+    display: flex;
+    flex-direction: row;
```
Comment (yashpatel1797) [inline page-css/wishlist.css]: Default value of flex-direction is row so no need to declare here.

**Review by yashpatel1797 — APPROVED** [review]
Added one suggestion else design and code Looks good to me.

**Comment by rishbitsnbytes** [comment]
Everything looks good.
Good to merge.

**Review by shuklahimanshu81 — APPROVED** [review]
Good to merge. Everything looks good.

**Pull request merged**